bin/hermes

```diff
@@ -32,7 +33,15 @@ def request_get(path):
     Returns:
         the http response
     """
-    response = requests.get(settings.hermes_server + path)
+    retries = 0
+    clear_response = False
+    while not clear_response and retries < 5:
```

Comment:
I'd expand on that since you have this in two places, an probably more in the future that this could be abstracted pretty quickly into a helper function with backoff and configurable sleep times rather than sleeping a solid second


Just an example:
```python
def retry(num_attempts, min_sleep=.2, max_sleep=1):
    for num in range(1, num_attempts+1):
        yield num

        # Don't sleep on final attempt. You've failed :(
        if num == num_attempts:
            break

        # Sleep longer for each attempt
        sleep_time = (min_sleep * (num * num))

        # Clamp the sleep time to max_sleep
        sleep_time = max(0, min(max_sleep, sleep_time))

        sleep(sleep_time)


for attempt in retry(5):
    response = requests.get(settings.hermes_server + path)
    if response.status_code < 500:
        break
```